src/glb_wdog_exec.c

```diff
@@ -174,7 +174,7 @@ exec_thread (void* arg)
         goto init_error;
     }
 
-    char* pargv[4] = { strdup ("sh"), strdup ("-c"), strdup (cmd), NULL };
+    char* pargv[4] = { strdup ("bash"), strdup ("-c"), strdup (cmd), NULL };
```

Comment:
I don't think that solution for a bug in `dash` should be hardcoding shell version in a program and thus adding extra package dependency. This is OS configuration issue. The user should be able to choose his shell as he likes.